src/language-server/server.ts

```diff
@@ -19,9 +19,17 @@ import {
   LanguageServerCommands as Commands,
   LanguageServerRequests as Requests,
 } from "../messages";
+import { isValidationError } from "zod-validation-error";
 
 const connection = createConnection(ProposedFeatures.all);
+
 Debug.SetConnection(connection);
+const { sendNotification: originalSendNotification } = connection;
+connection.sendNotification = async (...args: [any, ...any[]]) => {
+  await whenConnectionInitialized;
+  connection.sendNotification = originalSendNotification;
+  connection.sendNotification(...args);
+};
```

Comment:
All `sendNotification` calls before the LSP client was fully started were swallowed previously - this included `Debug.error` etc.
This queues them instead.